llvm/test/CodeGen/SystemZ/flag_output_operand_ccand.ll

```diff
@@ -0,0 +1,500 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; Test Flag Output Operands with 14 combinations of CCMASK and optimizations
+; for AND for 3 three different functions, including two test cases from heiko.
+
+; RUN: llc < %s -verify-machineinstrs -mtriple=s390x-linux-gnu -O2 | FileCheck %s
+
+; Test CC == 0 && CC == 1.
```

Comment:
This test seems a bit pointless.   `CC == 0 && CC == 1` is tautologically false, and that was apparently already recognized by common optimizers when you generated the test: see the `ret i32 0` IL statement.  So there's nothing left for this test to actually verify.  The same seems to hold for all the tests in this file.